src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1704,6 +1709,9 @@
                 this.options.onBlur.call(this, event);
             }
 
+            //reformat input - is always OK? or only if it was modified?
```

Comment:
I think its safe to remove this comment I think we should always format unless we find it to be a problem.